superset/explore/form_data/commands/create.py

```diff
@@ -44,18 +45,48 @@ def run(self) -> str:
             tab_id = self._cmd_params.tab_id
             actor = self._cmd_params.actor
             form_data = self._cmd_params.form_data
-            check_access(dataset_id, chart_id, actor)
-            contextual_key = cache_key(session.get("_id"), tab_id, dataset_id, chart_id)
-            key = cache_manager.explore_form_data_cache.get(contextual_key)
-            if not key or not tab_id:
-                key = random_key()
-            if form_data:
-                state: TemporaryExploreState = {
+            sl_id = self._cmd_params.sl_id
+            sl_type = self._cmd_params.sl_type
+
+            state: Optional[TemporaryExploreState] = None
+
+            if form_data and dataset_id:
+                check_access(dataset_id, chart_id, actor)
+                contextual_key = cache_key(
+                    session.get("_id"), tab_id, dataset_id, chart_id
+                )
+                state = {
                     "owner": actor.get_user_id(),
                     "dataset_id": dataset_id,
                     "chart_id": chart_id,
                     "form_data": form_data,
+                    "sl_id": None,
+                    "sl_type": None,
+                }
+
+            elif form_data and sl_type and sl_id:
```

Comment:
So, I think we're going to want to keep form_data cache or Queries, just not the data_cache (if it's simpler to exclude it). If we don't have form data cache, then people won't be able to send this page to anyone.